3_MindBridge_Databricks_E2E_Analysis_Data_Flow.py

```diff
@@ -26,7 +24,6 @@
 
 # COMMAND ----------
 
-import mindbridgeapi as mbapi
```

Comment:
For some reason removing the import from here causes the error
```NameError: name 'mbapi' is not defined```
Even though it is defined at the bottom of the previous block.

Reply:
It would be ideal to have import line in the same section as the install so that it forces users to use the most up to date version of the SDK.

I found this interesting - I switched %restart_python to dbutils.library.restartPython() and got the warning:
```Warning: statements after `dbutils.library.restartPython()` will execute before Python is restarted.```

Perhaps %restart_python works the same way, causing it to reset the session _after_ the import statement occurs, wiping it from the session.

Reply:
For this reason I think we need to keep the restart_python and the import in separate blocks.